### ISSUE TITLE
Suggestions for the output Movementscript

### ISSUE THREAD

**Issue by kinsi55** [KandyWrong/blender2camera2#2]
https://github.com/KandyWrong/blender2camera2/blob/137c9e1d08195b010a6912d2d844cd66ffff73cd/b2c2_export.py#L261

you should set duration instead of holdTime. holdTime dictates how long the script should freeze on this frames final values, duration dictates the time over which the camera will transition coming from the previous frame, so using the former, if you exported the script at 20 FPS, it would still be very smooth ingame due to interpolation in between frames.

Furthermore, you could consider only outputting the FOV if it actually changed coming from the previous frame to minimize script size 👍

**Comment by KandyWrong** [comment]
You're right, now that I look at it more closely I realize that definitely needs to be fixed. Thanks for pointing that out!

C'est la vie with software development, something always needs fixing 🙃

**Comment by KandyWrong** [comment]
I changed holdTime to duration in the v1.1 release. See #3 

As for the other bit about including changed values only when needed, as opposed to on every single frame, I'm gonna punt that to v1.2 which will happen sometime after my next major video release. 👍
